integrations/Podsights/constants.js

```diff
@@ -0,0 +1,160 @@
+const NAME = "PODSIGHTS";
+const CNameMapping = {
+  [NAME]: NAME,
+  Podsights: NAME,
+  PodSights: NAME,
+  "pod Sights": NAME,
+  "Pod Sights": NAME,
+  "pod sights": NAME,
+  "POD SIGHTS": NAME,
+  "Pod sights": NAME,
+};
+
+const LEAD_EVENT = [
+  {
+    destKey: "value",
+    sourceKeys: "value",
+  },
+  {
+    destKey: "currency",
+    sourceKeys: "currency",
+  },
+  {
+    destKey: "type",
+    sourceKeys: "type",
+  },
+  {
+    destKey: "category",
+    sourceKeys: "category",
+  },
+];
+
+const PRODUCT_EVENT = [
+  {
+    destKey: "value",
+    sourceKeys: "value",
+  },
+  {
+    destKey: "currency",
+    sourceKeys: "currency",
+  },
+  {
+    destKey: "product_id",
+    sourceKeys: "productId",
+  },
+  {
+    destKey: "product_name",
+    sourceKeys: "productName",
+  },
+  {
+    destKey: "product_type",
+    sourceKeys: "productType",
+  },
+  {
+    destKey: "product_vendor",
+    sourceKeys: "productVendor",
+  },
+];
+
+const ADD_TO_CART_EVENT = [
+  {
+    destKey: "value",
+    sourceKeys: "value",
+  },
+  {
+    destKey: "currency",
+    sourceKeys: "currency",
+  },
+  {
+    destKey: "product_id",
+    sourceKeys: "productId",
+  },
+  {
+    destKey: "product_name",
+    sourceKeys: "productName",
+  },
+  {
+    destKey: "product_type",
+    sourceKeys: "productType",
+  },
+  {
+    destKey: "product_vendor",
+    sourceKeys: "productVendor",
+  },
+  {
+    destKey: "quantity",
+    sourceKeys: "quantity",
+  },
+  {
+    destKey: "variant_id",
+    sourceKeys: "variantId",
+  },
+  {
+    destKey: "variant_name",
+    sourceKeys: "variantName",
+  },
+];
+
+const CHECK_OUT_EVENT = [
+  {
+    destKey: "value",
+    sourceKeys: "value",
+  },
+  {
+    destKey: "currency",
+    sourceKeys: "currency",
+  },
+  {
+    destKey: "discount_code",
+    sourceKeys: "discountCode",
+  },
+  {
+    destKey: "quantity",
+    sourceKeys: "quantity",
+  },
+  {
+    destKey: "line_items",
+    sourceKeys: "lineItems",
```

Comment:
same as the purchase event. If product array is not there, lets search from message.properties itself

Reply:
This event doesn't take any product properties, so we are not passing any product array.

Reply:
Same for this also, `line_items` doesn't have any effects in this event, So, As mentioned in the docs, should we take `line_items` object values from message.properties?

Reply:
do you see line_items not going in the network call?

Reply:
Just verified, line_items are going, not showing in the dashboard, but going on network calls. Also any other field, except what has been listed in the docs, are not going. So let us stick to the exact fields that they have said

Reply:
Okay, will update